modules/release-notes/pages/relnotes.adoc

```diff
@@ -4,6 +4,217 @@ Couchbase Server 6.6 adds features and enhancements to improve developer product
+| *Summary*: Starting with version 6.5.0, VbSeqnosReader has been updated to process two types of requests: VbSeqnosRequest and VbMinSeqnosRequest. When processing VbSeqnosRequest, if there are any VbMinSeqnosRequest's, then the VbMinSeqnosRequest's will be queued back into the requestCh of VbSeqnosReader. However, if the VbSeqnosReader closed by this time, then enqueue would fail and the caller would be waiting for a response indefinitely. This has been fixed to respond to outstanding requests upon exit of VbSeqnosReader.
+
+| https://issues.couchbase.com/browse/MB-42614[MB-42614^]
+| *Summary*: Fixed an issue where rebalance failed due to timestamp mismatch between snapshots.
+
+| https://issues.couchbase.com/browse/MB-42108[MB-42108^]
+| *Summary*: Fixed an issue where multiple partition tombstones for an index during rebalance could lead to partition cleanup on restart.
+
+| https://issues.couchbase.com/browse/MB-41722[MB-41722^]
+| *Summary*: Fixed an issue in the waitForIndexBuild routine which caused it not to terminate at the end of the batch and remain active till the end of rebalance. As a result, rebalance caused a very large number of TIME_WAIT connections and subsequently failed.
+
+| https://issues.couchbase.com/browse/MB-41673[MB-41673^]
+| *Summary*: Added per index `memory_used` statistic to the `api/v1/stats` endpoint to enable memory accounting.
+
+| https://issues.couchbase.com/browse/MB-41672[MB-41672^]
+| *Summary*: The statistic `pauseTotalNs` has been added to the `api/v1/stats` endpoint and enables you to monitor any spikes in GC between two intervals. `PauseTotalNs` is a cumulative statistic that represents the total time an indexer process has been paused for GC since it's inception.
+
+| https://issues.couchbase.com/browse/MB-41645[MB-41645^]
+| *Summary*: Fixed an issue where the gsi index resident ratio showed a value greater than 100% due to num_rec_swapin being larger than num_rec_swapout (num_rec_swapin > num_rec_swapout). This is a rare and transient condition that may occur sometimes as the stats are updated asynchronously and will become correct eventually.
+
+| https://issues.couchbase.com/browse/MB-41641[MB-41641^]
+| *Summary*: Improved array indexing performance by optimizing the ComputeArrayEntriesWithCount method.
+
+| https://issues.couchbase.com/browse/MB-41717[MB-41717^]
+| *Summary*: When bloomDelta is added after recovery when page is found without a bloom filter, the stat NumRecordAllocs is over counted. However, NumRecordAllocs is only supposed to track the insert/delete deltas. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-41155[MB-41155^]
+| *Summary*: Fixed an issue with memory optimized indexes where indefinite disk snapshotting led to increasing disk usage.
+
+| https://issues.couchbase.com/browse/MB-41091[MB-41091^]
+| *Summary*: The debugger link has been updated, from `chrome-devtools://` to `devtools://`,  to adapt to different Chrome versions.
```

Comment:
This issue should be listed in the Eventing section

Reply:
Good catch. Thanks!